pkg/console/starter/starter.go

```diff
@@ -5,6 +5,8 @@ import (
 	"os"
 	"time"
 
+	"github.com/openshift/library-go/pkg/operator/loglevel"
```

Comment:
nit:
let's put this to the other `openshift` libs

Reply:
done!